src/main/kotlin/no/nav/syfo/client/dokarkiv/domain/Dokument.kt

```diff
@@ -3,7 +3,7 @@ package no.nav.syfo.client.dokarkiv.domain
 enum class BrevkodeType(
     val value: String,
 ) {
-    FORESPORSEL_OM_PASIENT("OPPF_FORESP_OM_PAS"),
+    FORESPORSEL_OM_PASIENT("OPPF_FORESP_OM_PAS"), // TODO: Burde vi ha en egen brevkodetype for legeerklæring?
```

Comment:
Er ikke så kjent med brevkodene, men per nå klarer vi jo da ikke å skille på legeerklæringer og tilleggsopplysninger ved å beholde det slik?

Reply:
Usikker på om vi har behov for å skille på de, men kanskje det er viktig for joark/gosys. Hva tenker dere @JMLindseth @juneih - skal vi skille på brevkoder for legeeklæringer og tilleggsopplysninger?

Reply:
Vi sjekket i Gosys og disse brevkodene vises ikke der nå (fordi vi har funnet på de selv?). Hører med Stine senere om det er viktig å ha egne brevkoder for hver forespørsel-meldingtype.

Reply:
Vi hadde en diskusjon på det, og en sjekk i Gosys. De tidligere vi har lagt til vises ikke som brevkode der, så mulig vi må tipse Gosys om at det finnes nye koder. Det ligger ingen slike brev fra Arena i dev i dag, så vi ble enige om å sjekke hva som skjer i prod.